configs/mmcls/classification_ascend_dynamic-224x224-224x224.py

```diff
@@ -0,0 +1,9 @@
+_base_ = ['./classification_dynamic.py', '../_base_/backends/ascend.py']
+
+onnx_config = dict(input_shape=[224, 224])
+
+backend_config = dict(model_inputs=[
+    dict(
+        dynamic_batch_size=[1, 2, 4, 8],
```

Comment:
Does it mean the dynamic range of batch dim is [1, 8]?

Reply:
Actually, it's just 1, 2, 4 or 8, any other number will trigger an error in their API. We choose the smallest one suitable for input's batch size in our wrapper classes.